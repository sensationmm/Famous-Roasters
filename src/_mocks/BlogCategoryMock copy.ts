import { GraphQLError } from 'graphql'
import { loader } from 'graphql.macro'

const GET_BLOG_CATEGORY_LIST = loader('src/graphql/queries/blogCategoryListDetails.query.graphql')

export const BlogCategoryListMock = {
  request: {
    query: GET_BLOG_CATEGORY_LIST,
  },
  result: {
    data: {
      categoryList: [
        {
          __typename: 'Category',
          name: 'Kaffeewissen',
          summary: 'Werde zum Kaffee-Nerd. Unsere Experten erklären alles: Von Aroma bis Zubereitungsmethode.',
          tags: ['Nachhaltigkeit', 'SpecialityCoffee'],
        },
        {
          __typename: 'Category',
          name: 'Kaffeeequipment',
          summary:
            'Unsere Experten testen und bewerten das beste Kaffeeequipment. Wir wollen sicherstellen, dass du die beste Hardware für einen exzellenten Kaffee hast.',
          tags: ['Handfiltervergleich', 'Grundausstattung'],
        },
        {
          __typename: 'Category',
          name: 'Kaffeetrends',
          summary: 'Die aktuellsten Trends über die Kaffeewelt.',
          tags: ['Gadgets', 'Roestungen'],
        },
        {
          __typename: 'Category',
          name: 'Zubereitungstipps',
          summary:
            'Mach mehr aus deinem Kaffee: our experts prepared a few brewing guides to help you enjoy always the best coffee.',
          tags: ['Aeropress', 'Siebtraeger', 'ColdDrip'],
        },
      ],
    },
  },
}

export const BlogCategoryListMockError = {
  request: {
    query: GET_BLOG_CATEGORY_LIST,
  },
  result: {
    errors: [new GraphQLError('Error!')],
  },
}
