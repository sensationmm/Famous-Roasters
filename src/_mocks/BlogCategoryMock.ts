import { GraphQLError } from 'graphql'
import { loader } from 'graphql.macro'

const GET_BLOG_CATEGORY_LIST = loader('src/graphql/queries/blogCategoryListDetails.query.graphql')

export const BlogCategoryListMock = {
  request: {
    query: GET_BLOG_CATEGORY_LIST,
  },
  result: {
    data: {
      categoryList: [
        {
          __typename: 'Category',
          name: 'Kaffeewissen',
          summary:
            'Lever up your coffee-culture. Our expert will touch different topics: beans types, origins, cupping and many more...',
          tags: ['Nachhaltigkeit', 'SpecialityCoffee'],
        },
        {
          __typename: 'Category',
          name: 'Kaffeeequipment',
          summary:
            'Here our coffee experts test and review different equipment. They aim to cover from basic to more niche equipment.',
          tags: ['Handfiltervergleich', 'Grundausstattung'],
        },
        {
          __typename: 'Category',
          name: 'Kaffeetrends',
          summary:
            'Here our coffee experts test and review different equipment. They aim to cover from basic to more niche equipment.',
          tags: ['Gadgets', 'Roestungen'],
        },
        {
          __typename: 'Category',
          name: 'Zubereitungstipps',
          summary:
            'Mach mehr aus deinem Kaffee: our experts prepared a few brewing guides to help you enjoy always the best coffee.',
          tags: ['Aeropress', 'Siebtraeger', 'ColdDrip'],
        },
      ],
    },
  },
}

export const BlogCategoryListMockError = {
  request: {
    query: GET_BLOG_CATEGORY_LIST,
  },
  result: {
    errors: [new GraphQLError('Error!')],
  },
}
