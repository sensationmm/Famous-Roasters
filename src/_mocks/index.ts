import { CatalogueMockError, CatalogueMockMissingData, CatalogueMocks, CatalogueMocksPagination } from './CatalogueMock'
import { ProductMock } from './ProductMock'

export { CatalogueMocks as CatalogueMocks }
export { CatalogueMocksPagination as CatalogueMocksPagination }
export { CatalogueMockError as CatalogueMockError }
export { CatalogueMockMissingData as CatalogueMockMissingData }
export { ProductMock as ProductMock }
