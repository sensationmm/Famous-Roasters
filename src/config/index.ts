import { apolloClient } from './apollo'
import i18n from './i18n'

export { i18n as i18n }
export { apolloClient as apolloClient }
