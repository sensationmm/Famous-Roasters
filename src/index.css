@tailwind base;
@tailwind components;
@tailwind utilities;

body {
  margin: 0;
  padding: 0;
  font-family: 'Open Sans', sans-serif;
  font-size: 16px;
  line-height: 1.5em;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

body > div > div > main {
  min-height: 50vh;
}

:root {
  --swiper-theme-color: #646575 !important;
}

.video-container {
  position: relative;
  padding-bottom: 56.25%;
  padding-top: 0;
  height: 0;
  overflow: hidden;
}

.video-container iframe,
.video-container object,
.video-container embed {
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
}