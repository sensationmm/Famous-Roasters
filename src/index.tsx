import './index.css'

import React from 'react'
import { createRoot } from 'react-dom/client'

import App from './App'

const container = document.getElementById('root') as Element
const root = createRoot(container)
if (process.env.NODE_ENV === 'development') {
  root.render(
    <React.StrictMode>
      <App />
    </React.StrictMode>,
  )
} else {
  root.render(<App />)
}
