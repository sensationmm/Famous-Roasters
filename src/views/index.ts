import { About } from './About'
import { Account } from './Account'
import { Cart } from './Cart'
import { Catalogue } from './Catalogue'
import { Checkout } from './Checkout'
import { Contact } from './Contact'
import { Error } from './Error'
import { Home } from './Home'
import { OurRoasters } from './OurRoasters'
import { Product } from './Product'
import { TasteFinder } from './TasteFinder'

export { About as About }
export { Account as Account }
export { Cart as Cart }
export { Catalogue as Catalogue }
export { Checkout as Checkout }
export { Contact as Contact }
export { Error as Error }
export { Home as Home }
export { OurRoasters as OurRoasters }
export { Product as Product }
export { TasteFinder as TasteFinder }
