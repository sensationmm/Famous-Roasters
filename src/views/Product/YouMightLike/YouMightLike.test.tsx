import { MockedProvider } from '@apollo/client/testing'
import { render, waitFor } from '@testing-library/react'
import React from 'react'
import { I18nextProvider } from 'react-i18next'
import { MemoryRouter } from 'react-router-dom'
import { CatalogueMockAccessoriesYouMightLike, CatalogueMockRandom } from 'src/_mocks'
import { i18n } from 'src/config'

import { YouMightLike } from '.'

// eslint-disable-next-line @typescript-eslint/ban-ts-comment
// @ts-ignore
delete window.history
// eslint-disable-next-line @typescript-eslint/ban-ts-comment
// @ts-ignore
window.history = { go: jest.fn() }

jest.mock('react-router-dom', () => ({
  ...jest.requireActual('react-router-dom'), // use actual for all non-hook parts
  useParams: () => ({
    id: '7655228866776',
  }),
}))

describe('Find accessories partial view', () => {
  it('Renders correctly for a successful call', async () => {
    const { container } = render(
      <MockedProvider
        defaultOptions={{ watchQuery: { fetchPolicy: 'network-only' } }}
        mocks={[CatalogueMockAccessoriesYouMightLike, CatalogueMockRandom]}
        addTypename={false}
      >
        <I18nextProvider i18n={i18n}>
          <MemoryRouter initialEntries={['/product/7655228866776']}>
            <YouMightLike productId="123456789" />
          </MemoryRouter>
        </I18nextProvider>
      </MockedProvider>,
    )
    await waitFor(() => new Promise((res) => setTimeout(res, 500)))
    expect(container).toMatchSnapshot()
  })

  it('Renders correctly for a successful call not excluding', async () => {
    const { container } = render(
      <MockedProvider
        defaultOptions={{ watchQuery: { fetchPolicy: 'network-only' } }}
        mocks={[CatalogueMockAccessoriesYouMightLike, CatalogueMockRandom]}
        addTypename={false}
      >
        <I18nextProvider i18n={i18n}>
          <MemoryRouter initialEntries={['/product/7655228866776']}>
            <YouMightLike productId="7655228866776" />
          </MemoryRouter>
        </I18nextProvider>
      </MockedProvider>,
    )
    await waitFor(() => new Promise((res) => setTimeout(res, 500)))
    expect(container).toMatchSnapshot()
  })

  it('handles error', async () => {
    const { container } = render(
      <MockedProvider
        defaultOptions={{ watchQuery: { fetchPolicy: 'network-only' } }}
        mocks={[CatalogueMockAccessoriesYouMightLike]}
        addTypename={false}
      >
        <I18nextProvider i18n={i18n}>
          <MemoryRouter initialEntries={['/product/7655228866776']}>
            <YouMightLike productId="7655228866776" />
          </MemoryRouter>
        </I18nextProvider>
      </MockedProvider>,
    )
    await waitFor(() => new Promise((res) => setTimeout(res, 500)))
    expect(container).toMatchSnapshot()
  })

  it('Renders correctly for filtered to coffees version', async () => {
    const { container } = render(
      <MockedProvider
        defaultOptions={{ watchQuery: { fetchPolicy: 'network-only' } }}
        mocks={[CatalogueMockAccessoriesYouMightLike, CatalogueMockRandom]}
        addTypename={false}
      >
        <I18nextProvider i18n={i18n}>
          <MemoryRouter initialEntries={['/product/7655228866776']}>
            <YouMightLike productId="7655228866776" filter="coffee" />
          </MemoryRouter>
        </I18nextProvider>
      </MockedProvider>,
    )
    await waitFor(() => new Promise((res) => setTimeout(res, 500)))
    expect(container).toMatchSnapshot()
  })
})
