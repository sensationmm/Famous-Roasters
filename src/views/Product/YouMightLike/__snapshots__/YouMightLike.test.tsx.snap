// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Find accessories partial view Renders correctly for a successful call 1`] = `
<div>
  <div
    class="grid gap-2 grid-cols-1 md:grid-cols-2 xl:grid-cols-3"
  >
    <a
      href="/product/7655228866776"
    >
      <div
        class="flex pt-8"
      >
        <img
          alt="Espresso Raritäten Set"
          class="hidden"
          src="https://cdn.shopify.com/s/files/1/0632/7251/7848/products/ezgif-4-d921ab2e2b.png?v=1649246153"
        />
        <div
          class="flex justify-center items-center shrink-0 self-center relative w-32 h-32"
        >
          <div
            class="flex justify-center items-center rounded-full bg-coreUI-background-images w-32 h-32"
          >
            <img
              alt="Espresso Raritäten Set"
              class="w-28 max-h-28"
              src="https://cdn.shopify.com/s/files/1/0632/7251/7848/products/ezgif-4-d921ab2e2b.png?v=1649246153"
            />
            <div
              class="inline-flex rounded-full px-2 py-0.5 absolute top-2 left-0 bg-tags-santasGray text-white border border-white"
            >
              <span
                class="text-xs leading-4"
              >
                Decaf
              </span>
            </div>
          </div>
        </div>
        <div
          class="absolute"
        >
          <div
            class="flex justify-center items-center shrink-0 self-center relative w-32 h-32"
          >
            <div
              class="flex justify-center items-center rounded-full bg-coreUI-background-images w-32 h-32"
            >
              <div
                class="block bg-coreUI-text-tertiary rounded w-2/5 h-4/5"
              />
            </div>
          </div>
        </div>
        <div
          class="flex flex-col w-full justify-start pl-4"
        >
          <div
            class="text-base leading-6 font-semibold"
          >
            Espresso Raritäten Set
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            60beans
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            Kolumbien
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            Filter
          </div>
          <div
            class="flex items-baseline"
          >
            <span
              class="mr-1 text-base leading-6 font-semibold"
            >
              19,99 €
            </span>
            <span
              class="text-coreUI-text-secondary text-xs leading-4"
            >
              (
              10,00 €
              /kg)
            </span>
          </div>
        </div>
      </div>
    </a>
    <a
      href="/product/7655228866776"
    >
      <div
        class="flex pt-8"
      >
        <img
          alt="Espresso Raritäten Set"
          class="hidden"
          src="https://cdn.shopify.com/s/files/1/0632/7251/7848/products/ezgif-4-d921ab2e2b.png?v=1649246153"
        />
        <div
          class="flex justify-center items-center shrink-0 self-center relative w-32 h-32"
        >
          <div
            class="flex justify-center items-center rounded-full bg-coreUI-background-images w-32 h-32"
          >
            <img
              alt="Espresso Raritäten Set"
              class="w-28 max-h-28"
              src="https://cdn.shopify.com/s/files/1/0632/7251/7848/products/ezgif-4-d921ab2e2b.png?v=1649246153"
            />
            <div
              class="inline-flex rounded-full px-2 py-0.5 absolute top-2 left-0 bg-tags-santasGray text-white border border-white"
            >
              <span
                class="text-xs leading-4"
              >
                Decaf
              </span>
            </div>
          </div>
        </div>
        <div
          class="absolute"
        >
          <div
            class="flex justify-center items-center shrink-0 self-center relative w-32 h-32"
          >
            <div
              class="flex justify-center items-center rounded-full bg-coreUI-background-images w-32 h-32"
            >
              <div
                class="block bg-coreUI-text-tertiary rounded w-2/5 h-4/5"
              />
            </div>
          </div>
        </div>
        <div
          class="flex flex-col w-full justify-start pl-4"
        >
          <div
            class="text-base leading-6 font-semibold"
          >
            Espresso Raritäten Set
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            60beans
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            Kolumbien
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            Filter
          </div>
          <div
            class="flex items-baseline"
          >
            <span
              class="mr-1 text-base leading-6 font-semibold"
            >
              19,99 €
            </span>
            <span
              class="text-coreUI-text-secondary text-xs leading-4"
            >
              (
              10,00 €
              /kg)
            </span>
          </div>
        </div>
      </div>
    </a>
    <a
      href="/product/7655228866776"
    >
      <div
        class="flex pt-8"
      >
        <img
          alt="Espresso Raritäten Set"
          class="hidden"
          src="https://cdn.shopify.com/s/files/1/0632/7251/7848/products/ezgif-4-d921ab2e2b.png?v=1649246153"
        />
        <div
          class="flex justify-center items-center shrink-0 self-center relative w-32 h-32"
        >
          <div
            class="flex justify-center items-center rounded-full bg-coreUI-background-images w-32 h-32"
          >
            <img
              alt="Espresso Raritäten Set"
              class="w-28 max-h-28"
              src="https://cdn.shopify.com/s/files/1/0632/7251/7848/products/ezgif-4-d921ab2e2b.png?v=1649246153"
            />
            <div
              class="inline-flex rounded-full px-2 py-0.5 absolute top-2 left-0 bg-tags-santasGray text-white border border-white"
            >
              <span
                class="text-xs leading-4"
              >
                Decaf
              </span>
            </div>
          </div>
        </div>
        <div
          class="absolute"
        >
          <div
            class="flex justify-center items-center shrink-0 self-center relative w-32 h-32"
          >
            <div
              class="flex justify-center items-center rounded-full bg-coreUI-background-images w-32 h-32"
            >
              <div
                class="block bg-coreUI-text-tertiary rounded w-2/5 h-4/5"
              />
            </div>
          </div>
        </div>
        <div
          class="flex flex-col w-full justify-start pl-4"
        >
          <div
            class="text-base leading-6 font-semibold"
          >
            Espresso Raritäten Set
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            60beans
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            Kolumbien
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            Filter
          </div>
          <div
            class="flex items-baseline"
          >
            <span
              class="mr-1 text-base leading-6 font-semibold"
            >
              19,99 €
            </span>
            <span
              class="text-coreUI-text-secondary text-xs leading-4"
            >
              (
              10,00 €
              /kg)
            </span>
          </div>
        </div>
      </div>
    </a>
    <a
      href="/product/7655228866776"
    >
      <div
        class="flex pt-8"
      >
        <img
          alt="Espresso Raritäten Set"
          class="hidden"
          src="https://cdn.shopify.com/s/files/1/0632/7251/7848/products/ezgif-4-d921ab2e2b.png?v=1649246153"
        />
        <div
          class="flex justify-center items-center shrink-0 self-center relative w-32 h-32"
        >
          <div
            class="flex justify-center items-center rounded-full bg-coreUI-background-images w-32 h-32"
          >
            <img
              alt="Espresso Raritäten Set"
              class="w-28 max-h-28"
              src="https://cdn.shopify.com/s/files/1/0632/7251/7848/products/ezgif-4-d921ab2e2b.png?v=1649246153"
            />
            <div
              class="inline-flex rounded-full px-2 py-0.5 absolute top-2 left-0 bg-tags-santasGray text-white border border-white"
            >
              <span
                class="text-xs leading-4"
              >
                Decaf
              </span>
            </div>
          </div>
        </div>
        <div
          class="absolute"
        >
          <div
            class="flex justify-center items-center shrink-0 self-center relative w-32 h-32"
          >
            <div
              class="flex justify-center items-center rounded-full bg-coreUI-background-images w-32 h-32"
            >
              <div
                class="block bg-coreUI-text-tertiary rounded w-2/5 h-4/5"
              />
            </div>
          </div>
        </div>
        <div
          class="flex flex-col w-full justify-start pl-4"
        >
          <div
            class="text-base leading-6 font-semibold"
          >
            Espresso Raritäten Set
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            60beans
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            Kolumbien
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            Filter
          </div>
          <div
            class="flex items-baseline"
          >
            <span
              class="mr-1 text-base leading-6 font-semibold"
            >
              19,99 €
            </span>
            <span
              class="text-coreUI-text-secondary text-xs leading-4"
            >
              (
              10,00 €
              /kg)
            </span>
          </div>
        </div>
      </div>
    </a>
    <a
      href="/product/7655228866776"
    >
      <div
        class="flex pt-8"
      >
        <img
          alt="Espresso Raritäten Set"
          class="hidden"
          src="https://cdn.shopify.com/s/files/1/0632/7251/7848/products/ezgif-4-d921ab2e2b.png?v=1649246153"
        />
        <div
          class="flex justify-center items-center shrink-0 self-center relative w-32 h-32"
        >
          <div
            class="flex justify-center items-center rounded-full bg-coreUI-background-images w-32 h-32"
          >
            <img
              alt="Espresso Raritäten Set"
              class="w-28 max-h-28"
              src="https://cdn.shopify.com/s/files/1/0632/7251/7848/products/ezgif-4-d921ab2e2b.png?v=1649246153"
            />
            <div
              class="inline-flex rounded-full px-2 py-0.5 absolute top-2 left-0 bg-tags-santasGray text-white border border-white"
            >
              <span
                class="text-xs leading-4"
              >
                Decaf
              </span>
            </div>
          </div>
        </div>
        <div
          class="absolute"
        >
          <div
            class="flex justify-center items-center shrink-0 self-center relative w-32 h-32"
          >
            <div
              class="flex justify-center items-center rounded-full bg-coreUI-background-images w-32 h-32"
            >
              <div
                class="block bg-coreUI-text-tertiary rounded w-2/5 h-4/5"
              />
            </div>
          </div>
        </div>
        <div
          class="flex flex-col w-full justify-start pl-4"
        >
          <div
            class="text-base leading-6 font-semibold"
          >
            Espresso Raritäten Set
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            60beans
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            Kolumbien
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            Filter
          </div>
          <div
            class="flex items-baseline"
          >
            <span
              class="mr-1 text-base leading-6 font-semibold"
            >
              19,99 €
            </span>
            <span
              class="text-coreUI-text-secondary text-xs leading-4"
            >
              (
              10,00 €
              /kg)
            </span>
          </div>
        </div>
      </div>
    </a>
    <a
      href="/product/7655228866776"
    >
      <div
        class="flex pt-8"
      >
        <img
          alt="Espresso Raritäten Set"
          class="hidden"
          src="https://cdn.shopify.com/s/files/1/0632/7251/7848/products/ezgif-4-d921ab2e2b.png?v=1649246153"
        />
        <div
          class="flex justify-center items-center shrink-0 self-center relative w-32 h-32"
        >
          <div
            class="flex justify-center items-center rounded-full bg-coreUI-background-images w-32 h-32"
          >
            <img
              alt="Espresso Raritäten Set"
              class="w-28 max-h-28"
              src="https://cdn.shopify.com/s/files/1/0632/7251/7848/products/ezgif-4-d921ab2e2b.png?v=1649246153"
            />
            <div
              class="inline-flex rounded-full px-2 py-0.5 absolute top-2 left-0 bg-tags-santasGray text-white border border-white"
            >
              <span
                class="text-xs leading-4"
              >
                Decaf
              </span>
            </div>
          </div>
        </div>
        <div
          class="absolute"
        >
          <div
            class="flex justify-center items-center shrink-0 self-center relative w-32 h-32"
          >
            <div
              class="flex justify-center items-center rounded-full bg-coreUI-background-images w-32 h-32"
            >
              <div
                class="block bg-coreUI-text-tertiary rounded w-2/5 h-4/5"
              />
            </div>
          </div>
        </div>
        <div
          class="flex flex-col w-full justify-start pl-4"
        >
          <div
            class="text-base leading-6 font-semibold"
          >
            Espresso Raritäten Set
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            60beans
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            Kolumbien
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            Filter
          </div>
          <div
            class="flex items-baseline"
          >
            <span
              class="mr-1 text-base leading-6 font-semibold"
            >
              19,99 €
            </span>
            <span
              class="text-coreUI-text-secondary text-xs leading-4"
            >
              (
              10,00 €
              /kg)
            </span>
          </div>
        </div>
      </div>
    </a>
  </div>
</div>
`;

exports[`Find accessories partial view Renders correctly for a successful call not excluding 1`] = `
<div>
  <div
    class="grid gap-2 grid-cols-1 md:grid-cols-2 xl:grid-cols-3"
  >
    <a
      href="/product/7655228866776"
    >
      <div
        class="flex pt-8"
      >
        <img
          alt="Espresso Raritäten Set"
          class="hidden"
          src="https://cdn.shopify.com/s/files/1/0632/7251/7848/products/ezgif-4-d921ab2e2b.png?v=1649246153"
        />
        <div
          class="flex justify-center items-center shrink-0 self-center relative w-32 h-32"
        >
          <div
            class="flex justify-center items-center rounded-full bg-coreUI-background-images w-32 h-32"
          >
            <img
              alt="Espresso Raritäten Set"
              class="w-28 max-h-28"
              src="https://cdn.shopify.com/s/files/1/0632/7251/7848/products/ezgif-4-d921ab2e2b.png?v=1649246153"
            />
            <div
              class="inline-flex rounded-full px-2 py-0.5 absolute top-2 left-0 bg-tags-santasGray text-white border border-white"
            >
              <span
                class="text-xs leading-4"
              >
                Decaf
              </span>
            </div>
          </div>
        </div>
        <div
          class="absolute"
        >
          <div
            class="flex justify-center items-center shrink-0 self-center relative w-32 h-32"
          >
            <div
              class="flex justify-center items-center rounded-full bg-coreUI-background-images w-32 h-32"
            >
              <div
                class="block bg-coreUI-text-tertiary rounded w-2/5 h-4/5"
              />
            </div>
          </div>
        </div>
        <div
          class="flex flex-col w-full justify-start pl-4"
        >
          <div
            class="text-base leading-6 font-semibold"
          >
            Espresso Raritäten Set
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            60beans
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            Kolumbien
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            Filter
          </div>
          <div
            class="flex items-baseline"
          >
            <span
              class="mr-1 text-base leading-6 font-semibold"
            >
              19,99 €
            </span>
            <span
              class="text-coreUI-text-secondary text-xs leading-4"
            >
              (
              10,00 €
              /kg)
            </span>
          </div>
        </div>
      </div>
    </a>
    <a
      href="/product/7655228866776"
    >
      <div
        class="flex pt-8"
      >
        <img
          alt="Espresso Raritäten Set"
          class="hidden"
          src="https://cdn.shopify.com/s/files/1/0632/7251/7848/products/ezgif-4-d921ab2e2b.png?v=1649246153"
        />
        <div
          class="flex justify-center items-center shrink-0 self-center relative w-32 h-32"
        >
          <div
            class="flex justify-center items-center rounded-full bg-coreUI-background-images w-32 h-32"
          >
            <img
              alt="Espresso Raritäten Set"
              class="w-28 max-h-28"
              src="https://cdn.shopify.com/s/files/1/0632/7251/7848/products/ezgif-4-d921ab2e2b.png?v=1649246153"
            />
            <div
              class="inline-flex rounded-full px-2 py-0.5 absolute top-2 left-0 bg-tags-santasGray text-white border border-white"
            >
              <span
                class="text-xs leading-4"
              >
                Decaf
              </span>
            </div>
          </div>
        </div>
        <div
          class="absolute"
        >
          <div
            class="flex justify-center items-center shrink-0 self-center relative w-32 h-32"
          >
            <div
              class="flex justify-center items-center rounded-full bg-coreUI-background-images w-32 h-32"
            >
              <div
                class="block bg-coreUI-text-tertiary rounded w-2/5 h-4/5"
              />
            </div>
          </div>
        </div>
        <div
          class="flex flex-col w-full justify-start pl-4"
        >
          <div
            class="text-base leading-6 font-semibold"
          >
            Espresso Raritäten Set
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            60beans
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            Kolumbien
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            Filter
          </div>
          <div
            class="flex items-baseline"
          >
            <span
              class="mr-1 text-base leading-6 font-semibold"
            >
              19,99 €
            </span>
            <span
              class="text-coreUI-text-secondary text-xs leading-4"
            >
              (
              10,00 €
              /kg)
            </span>
          </div>
        </div>
      </div>
    </a>
    <a
      href="/product/7655228866776"
    >
      <div
        class="flex pt-8"
      >
        <img
          alt="Espresso Raritäten Set"
          class="hidden"
          src="https://cdn.shopify.com/s/files/1/0632/7251/7848/products/ezgif-4-d921ab2e2b.png?v=1649246153"
        />
        <div
          class="flex justify-center items-center shrink-0 self-center relative w-32 h-32"
        >
          <div
            class="flex justify-center items-center rounded-full bg-coreUI-background-images w-32 h-32"
          >
            <img
              alt="Espresso Raritäten Set"
              class="w-28 max-h-28"
              src="https://cdn.shopify.com/s/files/1/0632/7251/7848/products/ezgif-4-d921ab2e2b.png?v=1649246153"
            />
            <div
              class="inline-flex rounded-full px-2 py-0.5 absolute top-2 left-0 bg-tags-santasGray text-white border border-white"
            >
              <span
                class="text-xs leading-4"
              >
                Decaf
              </span>
            </div>
          </div>
        </div>
        <div
          class="absolute"
        >
          <div
            class="flex justify-center items-center shrink-0 self-center relative w-32 h-32"
          >
            <div
              class="flex justify-center items-center rounded-full bg-coreUI-background-images w-32 h-32"
            >
              <div
                class="block bg-coreUI-text-tertiary rounded w-2/5 h-4/5"
              />
            </div>
          </div>
        </div>
        <div
          class="flex flex-col w-full justify-start pl-4"
        >
          <div
            class="text-base leading-6 font-semibold"
          >
            Espresso Raritäten Set
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            60beans
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            Kolumbien
          </div>
          <div
            class="text-coreUI-text-secondary text-base leading-6"
          >
            Filter
          </div>
          <div
            class="flex items-baseline"
          >
            <span
              class="mr-1 text-base leading-6 font-semibold"
            >
              19,99 €
            </span>
            <span
              class="text-coreUI-text-secondary text-xs leading-4"
            >
              (
              10,00 €
              /kg)
            </span>
          </div>
        </div>
      </div>
    </a>
  </div>
</div>
`;

exports[`Find accessories partial view handles error 1`] = `
<div>
  <div
    class="p-4 border border-coreUI-border bg-white"
  >
    <div
      class="text-center text-base leading-6"
    >
      Ein Fehler ist aufgetreten
    </div>
    <div
      class="mt-4 flex justify-center"
    >
      <button
        class="relative inline-flex flex-row items-center rounded-full border-2 px-4 py-2 text-base font-bold disabled:bg-brand-grey-whisper disabled:text-coreUI-text-secondary disabled:cursor-not-allowed text-brand-black bg-primary border-transparent hover:bg-brand-green-starship active:bg-primary"
        data-testid="button-prompt"
        role="button"
      >
        Erneut versuchen
      </button>
    </div>
  </div>
</div>
`;
